src/FSharp.AWS.DynamoDB/TableContext.fs

```diff
@@ -320,9 +311,8 @@ type TableContext<'TRecord> internal
     /// </summary>
     /// <param name="item">Item to be written.</param>
     /// <param name="precondition">Precondition to satisfy in case item already exists.</param>
-    /// <param name="collector">Function to receive request metrics.</param>
-    member __.PutItemAsync(item : 'TRecord, precondition : Expr<'TRecord -> bool>, ?collector) = async {
-        return! __.PutItemAsync(item, template.PrecomputeConditionalExpr precondition, ?collector = collector)
+    member __.PutItemAsync(item : 'TRecord, precondition : Expr<'TRecord -> bool>) = async {
+        return! __.PutItemAsync(item, template.PrecomputeConditionalExpr precondition)
```

Comment:
the async return! layer ... is superfluous here?

Reply:
(I see some further similar cases further on too - pity I didnt see them when I touched all these lines before!)